.travis/deploy.sh

```diff
@@ -46,6 +49,9 @@ docker push makerdao/vdb-backfill-storage:$TAG
 message PUSHING BACKFILL-EVENTS DOCKER IMAGE
 docker push makerdao/vdb-backfill-events:$TAG
 
+message PUSHING EXTRACT-DIFFS DOCKER IMAGE
+docker push makerdao/vdb-extract-diffs:$TAG
```

Comment:
I wonder if we might also need to add a deployment script to this file below

Reply:
I'm also wondering if it would be good to remove the `if [ "$ENVIRONMENT" == "prod" ]; then` line here, since this is the beta branch. And then remove `elif [ "$ENVIRONMENT" == "staging" ]; then` from the prod branch?

Reply:
I like this idea! 💯

Reply:
great, i'll plan to do that new PRs